tuf/store/filestore.go

```diff
@@ -43,6 +43,10 @@ type FilesystemStore struct {
 func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 	fileName := fmt.Sprintf("%s.%s", name, f.metaExtension)
 	path := filepath.Join(f.metaDir, fileName)
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
But if we were talking about large files, `os.Stat` would be a lot more efficient than waiting for `ReadFile` to return.